src/services/user-admin.ts

```diff
@@ -89,9 +94,88 @@ export function deleteUsers(subs: string[]): Promise<void> {
 export function addUser(sub: string): Promise<void> {
     console.debug(`Creating sub user "${sub}"...`);
     return backendFetch(`${USER_ADMIN_URL}/users/${sub}`, { method: 'post' })
-        .then((response: Response) => undefined)
+        .then(() => undefined)
+        .catch((reason) => {
+            console.error(`Error while pushing the data : ${reason}`);
+            throw reason;
+        });
+}
+
+export type UserProfile = {
+    id: UUID | undefined;
+    name: string;
+    allParametersLinksValid: boolean | undefined;
+    loadFlowParameterId: UUID | undefined;
+};
+
+export function fetchProfiles(): Promise<UserProfile[]> {
```

Comment:
If you think it might be undefined we should explicit it here with UserProfile[] | undefined